docs-src/docs/reference/cli.md

```diff
@@ -512,6 +514,18 @@ l0 loadbalancer healthcheck [--set-target target | --set-interval interval |
 #### Additional information
 Calling the subcommand without flags will display the current configuration of the load balancer's health check. Setting any of the flags will update the corresponding field in the health check, and all omitted flags will leave the corresponding fields unchanged.
 
+### loadbalancer idletimeout
+Use the `idletimeout` subcommand to update the ELB Idle Timout of a load balancer.
```

Comment:
I assume it can be used to view the current idle timeout as well? Also [nit] `Timout`

Reply:
`l0 loadbalancer get` has now been modified to show the idle timeout of the load balancer. The subcommand `idletimeout` is just to update the loadbalancer's idle timeout.

Reply:
Although it should be noted that if the load balancer's idle timeout is updated, then the printer will print the load balancer with the new idle timeout value.

Reply:
ok, i think that's different behavior from the `healthcheck` subcommand, for example. I'm not sure if i agree that `idletimeout` deserves to be part of `loadbalancer get`.

Reply:
I think what I'm going to do is make it behave like `healthcheck` but I'd prefer leaving the `loadbalancer get` column that I added in.